.gitignore

```diff
@@ -130,4 +131,5 @@ build/builder-effective-config.yaml
 build/vcpkg_installed
 build/.cmake
 build/CMakeFiles
-build/CMakeCache.txt
\ No newline at end of file
+build/CMakeCache.txt
+.vscode/settings.json
```

Comment:
This was added to .gitignore, but also has pending changes getting pushed in this PR. Was that intentional?

Reply:
We can undo that yeh